Menu.java

```diff
@@ -0,0 +1,57 @@
+
+import java.util.Scanner;
+
+import DataOrganizer.DHT.DHT;
+ 
+public class Menu {
+ 
+ 
+    public static void informacja(){
+        System.out.println("Program do szyfrowania obrazów"
+                                + "jpeg z zastosowaniem instrukji switch.");
+    }
+ 
+    public static int menu(){
+        System.out.println();
+        System.out.println("     ****************************************");
+        System.out.println("     *                 MENU                 *");
+        System.out.println("     ****************************************");
+        System.out.println("     1. Encode");
+        System.out.println("     2. Decode");
+        System.out.println("     3. Informaja");
+        System.out.println("     0. Koniec");
+ 
+        Scanner in = new Scanner(System.in);
+        int w = in.nextInt();
+ 
+        return w;
+    }
+ 
+    public   Menu()
+    {
+        Scanner in = new Scanner(System.in);
+ 
+        int wybor = menu();
+ 
+        while(wybor!=0){
+            switch(wybor){
+                case 1:
+                    DHT.option= DHT.Option.ENCODE;
+                    break;
+                case 2:
+                    DHT.option= DHT.Option.DECODE;
+                    break;
+                case 3:
+                    informacja();
+ 
+            }
+ 
+ 
+            wybor = menu();
```

Comment:
Jak kod funkcji jest widoczny na jednej stronie bez scrollowania to się dużo lepiej go czyta. Dlatego powinny być krótkie i możliwie bez spacji. Taka drobna uwaga.